core/app/models/spree/state.rb

```diff
@@ -36,5 +36,9 @@ def <=>(other)
     def to_s
       name
     end
+
+    def state_with_country
+      "#{name}, (#{country})"
```

Comment:
I think we could remove the comma from here, WDYT?

Reply:
I'm fine with whatever, this is just a minor annoyance we noticed a couple of days ago. Whatever makes this easy to understand. 😄